### DISCUSSION TITLE
Conformal Prediction(for Hierarchical Time Series)

### DISCUSSION THREAD

**Discussion by katiavas** [valeman/awesome-conformal-prediction#37] [Q&A]
My question is  can I sum up conformal bounds so that they maintain their validity?
More context i needed: I am using  Ensemble batch prediction intervals (EnbPI) to output conformal upper and lower bounds for my time series data. I want to reconcile the upper bound and then the lower bound (input each one in the formula instead of y_pred) with MinTrace(or any otehr reconciliation method for that matter). Will this affect the validity coverage of my bounds or violate some assumptions? Or it is okay to sum up bounds like this ?
Here is the formula of MinTrace if it is of any help:
![image](https://github.com/valeman/awesome-conformal-prediction/assets/72811416/c15b94b9-b5de-4b46-8b3b-ebd03cddfc71)

**Comment by valeman** [comment]
This repo is not for questions, please use Awesome CP slack for this.
